app/components/GearListCard/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ *
+ * GearListCard
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+import GrayImageOverlay from 'components/shared/GrayImageOverlay';
+
+const ConstrainedImgBySize = styled.img`
+  width: 100%;
+  height: 8rem;
+`;
+
+const StyledNameOverLinkImage = styled.h3`
+  position: absolute;
+  bottom: 0;
+  left: 0.5rem;
```

Comment:
Wouldn't it be better to use margin here? Curious why not

Reply:
Heh, was just focusing on positioning the text as I was working. That makes sense.